src/adapters/autocomplete.js

```diff
@@ -28,30 +28,46 @@ function SearchInput(tagSelector) {
         this post is about correlation between gps coordinates decimal count & real precision unit
         110m = 3 decimals
        */
-      let promise = new Promise((resolve, reject) => {
+      let promise = new Promise(async (resolve, reject) => {
         /* 'bbox' is currently not used by the geocoder, it' will be used for the telemetry. */
-        let suggestHistoryPromise = StorePoi.get(term)
+        let suggestHistoryPromise = PoiStore.get(term)
         this.suggestPromise = BragiPoi.get(term)
-        Promise.all([this.suggestPromise, suggestHistoryPromise]).then((responses) => {
-          if(responses[0]) {
+        try {
+        let [bragiResponse, storeResponse] = await Promise.all([this.suggestPromise, suggestHistoryPromise])
+          if(bragiResponse) {
             this.suggestPromise = null
-            this.suggestList = responses[0].concat(responses[1])
+            this.suggestList = bragiResponse.concat(storeResponse)
             resolve(this.suggestList)
+          } else if(storeResponse) {
+            resolve(storeResponse)
```

Comment:
This breaks the abort mechanism in a subtle way  :fearful:
Currently, we rely on this promise to resolve to `null` when a suggest query is aborted. Since the `suggestPromise` is the only one to be aborted, it would now resolve to a list `storeResponse`. This list will be suggested anyway, and will cancel any pending `source` call ! Some typing in the autocomplete field will have no effect !  

I suppose we could fix this by resolving null, if any response is null:
```javascript
if (bragiResponse == null || storeResponse == null ){
    return resolve(null)
}
```

Or maybe we should clean up the `abort` mechanism to abort both the `suggestPromise` and the wrapping `Promise.all` ?